src/lib/Scenes/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -1,37 +1,64 @@
+import { Box, Join, Separator, Spacer } from "@artsy/palette"
 import { OtherWorks_artwork } from "__generated__/OtherWorks_artwork.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import { filter } from "lodash"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
-import { ArtworkContextArtistFragmentContainer as ArtworkContextArtist } from "./ArtworkContexts/ArtworkContextArtist"
+import { ContextGridCTA } from "./ContextGridCTA"
+import { Header } from "./Header"
 
-export const OtherWorksFragmentContainer = createFragmentContainer<{
-  artwork: OtherWorks_artwork
-}>(
+export const populatedGrids = grids => {
```

Comment:
We get back an array of grids from metaphysics, but it's possible that metaphysics will return a grid with 0 artworks (otherwise we'd have to make a bunch of extra fetches from within MP).

This helper method just returns the grids that have artworks. We reuse it in `Artwork.tsx` when deciding whether or not to push on the `OtherWorks` component at all.